config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -36,6 +36,38 @@ presubmits:
           - name: ndots
             value: "1"
 
+  - name: pull-cert-manager-make-test
+    always_run: false
+    optional: true
+    context: pull-cert-manager-make-test
+    max_concurrency: 8
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.8
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
+      description: Runs 'bazel test --jobs=1 //...'
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: 1.17.8-alpine@sha256:e2e68a9cdd5da82458652fdac3908a3a270686b38039f2829855398e2e06019d
```

Comment:
question: Is this image missing a repo? Is it `docker.io/library/golang`?

Reply:
Fixed in f69b0df, thanks.